config/local/config.yml

```diff
@@ -1,277 +1,284 @@
       signer_gas_balance:
         warning_threshold_wei: 250000000000000000
         critical_threshold_wei: 0
+      gas_price: <gas_price> # e.g. "30000000000"
+      min_gas_tip_cap: <min_gas_tip_cap> # e.g. 30000000000
+
   avalanche:
     chain_name: "avalanche"
     chain_id: "43114"
     type: "evm"
     environment: "mainnet"
     gas_token_symbol: "AVAX"
-    gas_token_coingecko_id: "avalanche-2"
     gas_token_decimals: 18
-    batch_uusdc_settle_up_threshold: 5000000
+    num_block_confirmations_before_fill: <num_block_confirmations_before_fill> # e.g. 1
+    batch_uusdc_settle_up_threshold: <batch_uusdc_settle_up_threshold> # e.g. 5000000
+    min_fill_size: <min_fill_size> # e.g. 5000000
+    max_fill_size: <max_fill_size> # e.g. 10000000000
+    max_rebalancing_gas_threshold: <max_rebalancing_gas_threshold> # e.g. 1000000
+    min_fee_bps: <min_fee_bps> # e.g. 100
+    solver_address: <solver_address> # e.g. "0x8EB49E3D65d74967CC0Fe987FA2d015ae816352E"
     evm:
-      rpc: <avalanche_rpc_server_url>
+      rpc: <avalanche_rpc_server_url> # e.g. "https://api.avax.network/ext/bc/C/rpc"
       rpc_basic_auth_var: <server_password>
       contracts:
         usdc_erc20_address: 0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E
       signer_gas_balance:
-        warning_threshold_wei: 1720000000000000000
-        critical_threshold_wei: 580000000000000000
-  avalanche-testnet:
-    chain_name: "avalanche"
-    chain_id: "43113"
-    type: evm
-    environment: "testnet"
-    gas_token_symbol: "AVAX"
-    gas_token_decimals: 18
-    evm:
-      rpc: <avalanche_testnet_rpc_server_url>
-      rpc_basic_auth_var: <server_password>
-      contracts:
-        usdc_erc20_address: 0x5425890298aed601595a70ab815c96711a31bc65
-      signer_gas_balance:
-        warning_threshold_wei: 1000000000000000000
-        critical_threshold_wei: 0
+        warning_threshold_wei: <warning_threshold_wei> # e.g. 1720000000000000000
+        critical_threshold_wei: <critical_threshold_wei> # e.g. 580000000000000000
+      gas_price: <gas_price> # e.g. "30000000000"
+      min_gas_tip_cap: <min_gas_tip_cap> # e.g. 30000000000
+
   optimism:
     chain_name: "optimism"
     chain_id: "10"
     type: evm
     environment: "mainnet"
     gas_token_symbol: "ETH"
-    gas_token_coingecko_id: "ethereum"
     gas_token_decimals: 18
+    num_block_confirmations_before_fill: <num_block_confirmations_before_fill> # e.g. 1
+    batch_uusdc_settle_up_threshold: <batch_uusdc_settle_up_threshold> # e.g. 5000000
+    min_fill_size: <min_fill_size> # e.g. 5000000
+    max_fill_size: <max_fill_size> # e.g. 10000000000
+    max_rebalancing_gas_threshold: <max_rebalancing_gas_threshold> # e.g. 1000000
+    min_fee_bps: <min_fee_bps> # e.g. 100
+    solver_address: <solver_address> # e.g. "0x8EB49E3D65d74967CC0Fe987FA2d015ae816352E"
     evm:
-      rpc: <optimism_rpc_server_url>
+      rpc: <optimism_rpc_server_url> # e.g. "https://mainnet.optimism.io"
```

Comment:
should this be specific to optimism? or boradly an evm rpc? arbitrum, base etc.

Reply:
specific to optimism